src/cmd_processor.cc

```diff
@@ -0,0 +1,46 @@
+#include "cmd_processor.h"
+
+#include <libqalculate/Calculator.h>
+
+namespace bb
+{
+    class CMDProcessor::Impl
+    {
+    public:
+        Impl();
+        std::string process(const std::string &cmd);
+
+        Calculator calc;
+    };
+}
+
+// Internal implementation
+
+bb::CMDProcessor::Impl::Impl()
+    : calc(Calculator())
```

Comment:
Since you are calling the default constructor and allocating in the stack, the initialization is not required. In fact here you are calling the copy constructor not the default constructor

Reply:
fixed! 7991eb7